scss/_settings_spacing.scss

```diff
@@ -31,7 +31,7 @@ $nudges: (
   nudge--h4: 0.05rem,
```

Comment:
Remove these leading zeroes

Reply:
I thought they were introduced because of prettier by @barrymcgee? At least that's what the commit message says.

Reply:
@deadlight @lyubomir-popov Yes, we now enforce leading zeros after adopting Prettier.